### PR TITLE
db: Postgres docker and CreateUsers query.

### PR DESCRIPTION
**Pull request by m-kuzmin**
# Changelog

- Created first migration
- Added SQLC for SQL queries
- Added commands to Makefile to setup PostgreSQL in docker
- Updated CI to check SQLC code.

### CODE REVIEW

**Comment by m-kuzmin** [comment]
I want to test the new CI config so I'll temporarily make it "ready".

**Comment by m-kuzmin** [comment]
It seems the CI config is ok. I will squash the ci file related commits and it should trigger a CI run too.

**Comment by m-kuzmin** [comment]
Welp, I wish there was a button to just run the CI on this branch because if it fails the only thing you can do is do one of the triggers such as marking PR as draft etc. So annoying.

**Comment by m-kuzmin** [comment]
Finally fixed the CI I will squash the messy history then...

File: main.go [hunk 1]
```diff
@@ -1,21 +1,105 @@
...
+
 	"github.com/gin-gonic/gin"
+	_ "github.com/golang-migrate/migrate/source/file"
+	_ "github.com/lib/pq"
 	"github.com/m-kuzmin/simple-rest-api/api"
 	"github.com/m-kuzmin/simple-rest-api/db"
 	"github.com/m-kuzmin/simple-rest-api/logging"
 )
 
+const (
```
Comment (penkova) [inline main.go:21]: these are default values? How about move it to config file?

**Review by penkova — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (m-kuzmin) [reply main.go:21]: I will add this in the next pr

**Pull request merged**